examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -254,6 +266,28 @@ void DeviceCallbacks::OnColorControlAttributeChangeCallback(EndpointId endpointI
 }
 #endif
 
+void DeviceCallbacks::OnIdentifyPostAttributeChangeCallback(EndpointId endpointId, AttributeId attributeId, uint16_t size,
+                                                            uint8_t * value)
+{
+    if (attributeId == Clusters::Identify::Attributes::IdentifyTime::Id)
+    {
+        uint16_t identifyTime;
+        size == 2 ? identifyTime = (value[1] << 8) + value[0] : identifyTime = value[0];
+        if (identifyTime)
+        {
+            // Currently we have no separate indicator LEDs on each endpoints.
+            // We are using LED1 for endpoint 0,1 and LED2 for endpoint 2
+            endpointId == 2 ? statusLED2.Blink(kIdentifyTimerDelayMS * 2) : statusLED1.Blink(kIdentifyTimerDelayMS * 2);
+        }
+        else
+        {
+            bool onOffState;
+            endpointId == 0 ? onOffState = mEndpointOnOffState[0] : onOffState = mEndpointOnOffState[endpointId - 1];
```

Comment:
Might be more readable as:
 ```suggestion
            bool onOffState = mEndpointOnOffState[endpointId == 0 ? 0 : endpointId - 1];
```
but don't feel too strongly about it.